packages/components/src/components/Menu/Menu.tsx

```diff
@@ -50,14 +50,15 @@ const Menu = React.forwardRef<HTMLButtonElement, MenuProps>(
 
         <AriakitMenu state={state}>
           <Box.div
-            borderColor="colorBorderPrimary"
+            backgroundColor="colorBackground"
             borderRadius="borderRadius20"
-            borderStyle="borderStyleSolid"
-            borderWidth="borderWidth10"
             display="flex"
             flexDirection="column"
-            paddingBottom="space20"
-            paddingTop="space20"
+            maxWidth="280px"
```

Comment:
Late to the party here but shouldn't this maxWidth be configurable as a prop? If we have text a bit bigger as option will the line break?

Reply:
Thank you for the heads up, Rafa. 🙏🏻 
I spoke with Hanna, who said the option should break the line when reaching the maxWidth. 

<img width="428" alt="image" src="https://github.com/Localitos/pluto/assets/5320963/ea4ec781-c84c-4e4f-95c5-14a866de8151">